src/cider/exec/nextgen/CMakeLists.txt

```diff
@@ -35,4 +35,4 @@ add_library(
   nextgen STATIC
   Nextgen.cpp $<TARGET_OBJECTS:cider_operators> $<TARGET_OBJECTS:cider_context>
   $<TARGET_OBJECTS:cider_parsers> $<TARGET_OBJECTS:cider_transformer>)
-target_link_libraries(nextgen ${NEXTGEN_DEPS} jitlib)
+target_link_libraries(nextgen ${NEXTGEN_DEPS} jitlib folly)
```

Comment:
why need folly? maybe link hashtable lib would be enough?

Reply:
folly is too heavy， please try to avoid using third-party lib in nextgen

Reply:
hashtable's impl will move outside h file to cpp file in future PR and will be a library. So nextgen will not use folly by then